services/common/src/components/projectSummary/FacilityOperator.tsx

```diff
@@ -1,56 +1,26 @@
-import React, { FC, useEffect, useState } from "react";
+import React, { FC } from "react";
 import { useSelector } from "react-redux";
 import { Col, Row, Typography } from "antd";
-import { Field, getFormValues, getFormSyncErrors } from "redux-form";
+import { Field, getFormValues } from "redux-form";
 import {
   email,
   maxLength,
   phoneNumber,
   postalCodeWithCountry,
   required,
   requiredRadioButton,
-  lat,
-  lon,
-  max,
-  min,
 } from "@mds/common/redux/utils/Validate";
 import RenderField from "../forms/RenderField";
 import RenderSelect from "../forms/RenderSelect";
 import { FORM } from "../..";
 import { getDropdownProvinceOptions } from "@mds/common/redux/selectors/staticContentSelectors";
 import RenderRadioButtons from "../forms/RenderRadioButtons";
 import RenderAutoSizeField from "../forms/RenderAutoSizeField";
-import CoreMap from "../common/Map";
 import { normalizePhone } from "@mds/common/redux/utils/helpers";
 
 export const FacilityOperator: FC = () => {
   const formValues = useSelector(getFormValues(FORM.ADD_EDIT_PROJECT_SUMMARY));
-  const formErrors = useSelector(getFormSyncErrors(FORM.ADD_EDIT_PROJECT_SUMMARY));
```

Comment:
The figma mockup had the removed inputs of the facility section setup within the legal land section (now called Location, Access and Land Use). So the work here is just removing them.